packages/react-cosmos-webpack/src/build-module-paths.js

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs';
+import glob from 'glob';
+
+// Modules are require calls wrapped in functions instead of plain paths
+// in order to help bundlers like Webpack know which files to bundle.
+const getRequireCb = (path) => `()=>require('${path}')`;
+
+const getFixtures = (componentsRootPath, componentCleanPath) => {
+  const fixturesRootPath = `${componentsRootPath}/__fixtures__/${componentCleanPath}`;
+
+  let fixtures = '{';
+  glob.sync(`${fixturesRootPath}/*.{js,json}`).forEach((fixturePath) => {
+    const nameMatcher = new RegExp(`^${fixturesRootPath}/(.+).(js|json)$`);
+    const matches = fixturePath.match(nameMatcher);
+    const name = matches[1];
+    const ext = matches[2];
```

Comment:
Maybe [path.extname](https://nodejs.org/api/path.html#path_path_extname_path)  along with [path.basename](https://nodejs.org/api/path.html#path_path_basename_path_ext) can ease the work done here.